groups/live/profiles/heritage-live-eu-west-2/vars

```diff
@@ -54,11 +54,12 @@ jfil_cw_logs = {
 }
 
 # bulk Gateway Server
-bulk_gateway_application        = "Bulk Gateway Server"
-bulk_gateway_ec2_name           = "bulk-gateway"
+bulk_gateway_application        = "bulk-gateway"
 bulk_gateway_ec2_instance_type  = "m5.large"
 bulk_gateway_ami                = "amzn2-base-*"
+bulk_gateway_instance_count     = 1
 bulk_gateway_bck_retention_days = "backup21"
+cloudwatch_namespace            = "BULKGATEWAY"
```

Comment:
Change this to `CUSTOM/SFTP`

Reply:
Changed to cloudwatch_namespace            = "CUSTOM/SFTP"